### ISSUE TITLE
Can't set order to 0 in Corrfunc.bases.spline

### ISSUE THREAD

**Issue by abbyw24** [kstoreyf/suave#2]
```
proj_type = 'generalr'
kwargs = {'order': 0} # 1: linear spline
projfn = 'quadratic_spline.dat'
nprojbins = int(nbins/1)
spline.write_bases(rmin, rmax, nprojbins, projfn, ncont=1000, **kwargs)
```
returns
```
No order given, defaulting to 1 (linear)
```

**Comment by abbyw24** [comment]
I think the problem is here
```
def get_bases(rmin, rmax, nbins, order=None, ncont=1000):
    if not order:
        print("No order given, defaulting to 1 (linear)")
```
this appears [here](https://github.com/kstoreyf/Corrfunc/blob/d9c821bcebd7225cf43ec9e09dfe817387c73f62/Corrfunc/bases/spline.py#L37)

**Comment by kstoreyf** [comment]
fixed in #3 (f8fdab3)

**Comment by kstoreyf** [comment]
fixed in #3 (f8fdab3)
